tests/reducers/fetchOrdersReducer.test.js

```diff
@@ -17,65 +18,144 @@ describe('fetchorders reducer test-suite', () => {
   };
   it(`sets status key loading to true upon action type
   FETCH_ORDERS_LOADING`, () => {
-    expect(fetchOrdersReducer(
-      initialState,
-      {
-        type: FETCH_ORDERS_LOADING,
-      }
-    )).toEqual(
-      {
-        ...initialState,
-        status: {
-          ...initialState.status,
-          loading: true,
-        }
-      }
-    )
+    expect(fetchOrdersReducer(initialState, {
+      type: FETCH_ORDERS_LOADING,
+    })).toEqual({
+      ...initialState,
+      status: {
+        ...initialState.status,
+        loading: true,
+      },
+    });
   });
 
   it(`sets status key fetched to true and parses data from payload
   to Orders key on FETCH_ORDERS_SUCCESS`, () => {
+    expect(fetchOrdersReducer(initialState, {
+      type: FETCH_ORDERS_SUCCESS,
+      data: [1, 2, 3],
+    })).toEqual({
+      ...initialState,
+      orders: [1, 2, 3],
+      status: {
+        ...initialState.status,
+        fetched: true,
+      },
+    });
+  });
+
+  it(`sets status key error to true and parses errormessage from
+  payload to  key errorMessage on FETCH_ORDERS_FAILURE`, () => {
+    expect(fetchOrdersReducer(initialState, {
+      type: FETCH_ORDERS_FAILURE,
+      payload: 'an error occured',
+    })).toEqual({
+      ...initialState,
+      errorMessage: 'an error occured',
+      status: {
+        ...initialState.status,
+        error: true,
+        loading: false,
+      },
+    });
+  });
+
+  it('filters the data in the results array upon action type SORT_AND_FILTER', () => {
     expect(fetchOrdersReducer(
-      initialState,
       {
-        type: FETCH_ORDERS_SUCCESS,
-        data: [ 1, 2, 3],
-      }
-    )).toEqual(
+        ...initialState,
+        orders: {
+          results: [
+            {
+              id: 1,
+              type: 'drugorder',
+            },
+            {
+              id: 2,
+              type: 'testorder',
+            },
+          ],
+        },
+      },
+      {
+        type: 'SORT_AND_FILTER',
+        sortBy: 'type',
+        value: 'drugorder',
+      },
+    )).toEqual({
+      ...initialState,
+      orders: {
+        results: [
+          {
+            id: 1,
+            type: 'drugorder',
+          },
+        ],
+      },
+    });
+  });
+
+  it('uses the data in localStorage if the filter array is empty', () => {
+    expect(fetchOrdersReducer(
       {
         ...initialState,
-        orders: [1, 2, 3],
-        status: {
-          ...initialState.status,
-          fetched: true,
-        }
-      }
-    )
+        orders: {
+          results: [],
+        },
+      },
+      {
+        type: 'SORT_AND_FILTER',
+        sortBy: 'type',
+        value: 'drugorder',
+      },
+    )).toEqual({
+      ...initialState,
+      orders: {
+        results: [],
+      },
```

Comment:
This test suit states
```
it('uses the data in localStorage if the filter array is empty'...
```
But I don't seem to find anywhere in this test suit that the `localStorage` is used, am I missing something?

Reply:
addressed